### DISCUSSION TITLE
Only use uv for some factors

### DISCUSSION THREAD

**Discussion by Cadair** [tox-dev/tox-uv#135] [Q&A]
Hello,

I am interested in using uv for some factors and not others.

Initially, I tried to make the `requires =` line factor specific but that didn't seem to work, next I tried to do this:

```
runner =
    !oldestdeps: virtualenv
    oldestdeps: uv-venv-runner
package_tox_env_type =
    !oldestdeps: virtualenv-pep-517
    oldestdeps: uv-venv-pep-517
```

but that ends up throwing this error:
```
ROOT: HandledError| .pkg is already defined as a uv-venv-pep-517, cannot be virtualenv-pep-517 too
```

It seems that there's not a way to revert the overriding of the default by tox-uv which works? or am I missing something obvious?!

**Comment by ssbarnea** [comment]
I totally agree with you, especially as I am sure that we will keep finding blocking bugs caused by either this plugin or uv itself, so having an escape route would be very useful.
